microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1302,6 +1305,360 @@
         }
     }
 
+  function createDiversityGaugeChart(diversityInSampleValue) {
+        const width = 500;
+        const height = 310;
+        const radius = Math.min(width, height) / 2;
+
+        const svg = d3.select("#diversity-gauge-chart")
+            .append("svg")
+            .attr("width", width)
+            .attr("height", height)
+            .append("g")
+            .attr("transform", `translate(${(width / 2) - 35}, ${height / 2})`);
+
+        const arc = d3.arc()
+            .innerRadius(radius * 0.57)
+            .outerRadius(radius * 0.82)
+            .startAngle(Math.PI * 0.75)
+            .endAngle(Math.PI * 2.15);
+
+        svg.append("path")
+            .attr("d", arc)
+            .attr("transform", `translate(-27.5, 15) rotate(100)`)            
+            .style("fill", "#D9E9EF")
+            .style("stroke", "#D9E9EF")
+            .style("stroke-width", 2);
+
+        svg.append("image")
+            .attr("xlink:href", "/static/img/Line_38.svg")
```

Comment:
Instead of importing SVGs for the lines, it makes more sense to use d3 to draw them. This applies to all of the instances of line SVGs.